docs/getting-started/detections-req.asciidoc

```diff
@@ -66,9 +68,20 @@ a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for t
 |The `manage` privilege
 a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices:
 
+* `.alerts-security.alerts-*`
 * `.siem-signals-*`
 * `.lists-*`
 * `.items-*`
+
+*Note*: As of {stack} version 8.0.0 and newer, the `.siem-signals-*` index has been renamed to `.alerts-security.alerts-*`. If you are upgrading to 8.0.0, it is recommended that users have privileges to both indices. If you are newly installing the {stack}, then you don't need privileges to the `.siem-signals-*` index.
```

Comment:
This is currently only included in this particular row of the table, but it seems just as relevant to all the other rows where we've added the new index name. Maybe move it above the table to be a more global note?